recipes/force-delete-win/meta.yaml

```
@@ -0,0 +1,60 @@
+{% set name = "force-delete-win" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name.replace('-', '_') }}/{{ name.replace('-', '_') }}-{{ version }}.tar.gz
+  sha256: d00fcc3f6266ea52668839fcb941a6aa2fe13854a94e96bf9b65918ca6d90607
+
+build:
+  skip: true  # [not win]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - clang
+    - llvm
+    - rust
+    - llvmdev
+    - llvm-tools
+    - libclang
+    - maturin
+    - toml
```

These should go into `host`
```suggestion
    - maturin
    - toml
```

---

Maturin is not needed at run-time, thus is should be a build-only dependency

---

Yes, but it is a build-only dependency in the host environment.